### ISSUE TITLE
Remove hard coded ohara version from manager

### ISSUE THREAD

**Issue by konekoya** [oharastream/ohara#1893]
ref #1867

**Comment by chia7712** [comment]
there is already a version file in manager image. the content is shown below.

```
ohara 0.7.0-SNAPSHOT
revision=6468315d7fdd8e71f659893713aa929f038ed2d2
Compiled by Chia-Ping Tsai on 2019-07-16 15:00:28
```

and the file is located at ${OHARA_HOME}/bin/ohara_version.

However, the file does not exist in either source code or testing.

As a solution, I prefer to generate a js code with version details through gradle. The generated file is able to appears in source/testing/production.

**Comment by konekoya** [comment]
Yep. So I plan to remove this version file and need your help in #1888. With the new generated file we will be able to display the version info across all the environments (source/testing/production) :)

**Comment by konekoya** [comment]
Resolved in #3425
